lib/Driver/UnixToolChains.cpp

```diff
@@ -42,8 +42,7 @@ std::string
 toolchains::GenericUnix::sanitizerRuntimeLibName(StringRef Sanitizer,
                                                  bool shared) const {
   return (Twine("libclang_rt.") + Sanitizer + "-" +
-          this->getTriple().getArchName() +
-          (this->getTriple().isAndroid() ? "-android" : "") + ".a")
```

Comment:
I had [added this several years ago](https://github.com/apple/swift/pull/27167/files#r353616826), but it is no longer needed: see apple/swift-driver#1372 for more info.